modules/consensus/synchronize.go

```diff
@@ -10,11 +10,16 @@ import (
 	"github.com/NebulousLabs/bolt"
 )
 
-const (
+var (
 	// MaxCatchUpBlocks is the maxiumum number of blocks that can be given to
 	// the consensus set in a single iteration during the initial blockchain
 	// download.
-	MaxCatchUpBlocks = 10
+	MaxCatchUpBlocks = func() types.BlockHeight {
+		if build.Release == "testing" {
+			return 3
+		}
+		return 10
```

Comment:
we prefer to have all units explicitly stated. if build.Release == "dev", "standard", and then default to `panic()`